product_filters/models/product.py

```diff
@@ -1,4 +1,5 @@
 # coding: utf-8
+# pylint: disable=expression-not-assigned
```

Comment:
Otherwise, you could use `COALESCE` from your sql query.

E.g. of a `COALESCE` `psql postgres -c "SELECT COALESCE('myval','default_if_Null1') UNION SELECT COALESCE(Null,'default_if_Null2')"`
Result: `myval, default_if_Null2`